CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.14)
+
+project(XPanel VERSION 1.3 LANGUAGES CXX)
+
+string(TOLOWER ${PROJECT_NAME} PROJECT_NAME_LOWERCASE)
+
+list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/.cmake)
+
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
+set(CMAKE_CXX_STANDARD 20 CACHE STRING "C++ standard")
+set(CMAKE_CXX_STANDARD_REQUIRED ON CACHE BOOL "C++ standard is a requirement")
+set(CMAKE_CXX_EXTENSIONS OFF CACHE BOOL "C++ compiler specific extensions")
+
+if (NOT MSVC)
+    add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+include(CTest)
+
+add_subdirectory(hidapi)
+add_subdirectory(LUA)
+add_subdirectory(SDK)
+
+add_subdirectory(src)
+add_subdirectory(test)
+
```

Comment:
on Windows it is more convenient to copy the lua and hidapi dll to the output folder.

on all plataform the board-config.ini is also needed if you use an own HW (like an Arduino board)

I'm not very familiar with CMake's syntax but something like that could help:

install(FILES config/board-config.ini DESTINATION XPanel/64)
if (WIN32)
install(FILES lua/lua54.dll DESTINATION XPanel/64)
install(FILES hidapi/lib/hidapi.dll DESTINATION XPanel/64)
endif()

Reply:
Fixed. Thanks.